### ISSUE TITLE
Question: Parameters in routes?

### ISSUE THREAD

**Issue by basickarl** [websockets/ws#1610]
I'm wondering is ws has route parameters inbuilt: If you take a look at the following express routing page (https://expressjs.com/en/guide/routing.html) in the section "Route parameters" you are able to do things such as "/users/:id" where id is put into the req.params object. 

If ws had this it would make it incredibly powerful since people would then be able to build http / websocket apis side by side.

**Comment by lpinca** [comment]
No, it's out of the scope of the module. Take a look at https://github.com/HenningM/express-ws.
